### PR TITLE
Orderers cleanup

### PR DESCRIPTION
**Pull request by ranchalp**
Refactor code (a bit more)

### CODE REVIEW

**Comment by ranchalp** [comment]
Anticipating your potential comment, I moved as much as possible from proto messages to go, but I figured that if some proto message must remain (because of it being used as an actual event) then it is better to leave them as the context of `hasherpbdsl` and `cryptopbdsl` `UponResult` methods, than to create new structs with the same structure as the proto messages (less boilerplate).

File: pkg/orderers/pbftprotobufs.go [hunk 1]
```diff
@@ -3,61 +3,18 @@ Copyright IBM Corp. All Rights Reserved.
...
-}
-
-func catchUpResponseHashOrigin(preprepare *pbftpb.Preprepare) *ordererpb.HashOrigin_Pbft {
-	return &ordererpb.HashOrigin_Pbft{
-		Pbft: &pbftpb.HashOrigin{Type: &pbftpb.HashOrigin_CatchUpResponse{CatchUpResponse: preprepare}},
-	}
-}
-
 // ============================================================
 // Serialization
```
Comment (matejpavlovic) [inline pkg/orderers/pbftprotobufs.go:19]: Consider renaming this file, as it is only about serialization now, not about protobufs.

**Review by matejpavlovic — APPROVED** [review]
Amazing, the code is way cleaner and simpler, _and_ there is less of it!

File: pkg/orderers/pbftprotobufs.go (hunk 1 above)
Comment (ranchalp) [reply pkg/orderers/pbftprotobufs.go:19]: Done!

**Pull request merged**